lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -171,6 +149,19 @@ def get_position_runtime(self, ax, renderer):
         else:
             return self._locator(ax, renderer).bounds
 
+    @staticmethod
+    def _calc_k(sizes, total):
+        # sizes is a (n, 2) array of (rel_size, abs_size); this method finds
+        # the k factor such that sum(rel_size * k + abs_size) == total.
+        rs_sum, as_sum = sizes.sum(0)
+        return (total - as_sum) / rs_sum if rs_sum else 0
```

Comment:
While at it, let's use better names
```suggestion
        rel_sum, abs_sum = sizes.sum(0)
        return (total - abs_sum) / rel_sum if rel_sum else 0
```

Reply:
sure, done throughout.